docs/pages/dsCompositeIn/ds-compositeIn-High-Level_TestSpec.md

```diff
@@ -169,3 +170,24 @@ Emulator will boot with the port informations coming from the configuration file
 ### Control Plane Requirement - Callback for Status Change
 
 Connecting and disconnecting source devices in the CompositeIn will be handled by the Control Plane.
+
+
+## Callback for video mode change
+
+|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----------|--------|--|--|--------------------------|
+|Verify that the callback function properly updates the video mode changes and notifies the application when a COMPOSITE Input video mode changes after the composite signal is stable |dsCompositeInRegisterVideoModeUpdateCB|N|Y|Y|
+
+### Test Startup Requirement - Callback for video mode change
+
+Launching of the source device with the CompositeIn connected.
+
+### Emulator Requirement - Callback for video mode change
+
+Emulator will boot with the port information and its video mode changes coming from the configuration file.
```

Comment:
make it as "and its video port change from the configuration file"

Reply:
Addressed. Please review

Reply:
remove the word coming